go.mod

```diff
@@ -45,7 +43,11 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.24.4
 	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.24.4
 	k8s.io/metrics => k8s.io/metrics v0.24.4
-	k8s.io/mount-utils => k8s.io/mount-utils v0.24.4
+	k8s.io/mount-utils => k8s.io/mount-utils v0.24.7-rc.0
 	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.24.4
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.24.4
 )
+
+replace k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.24.4
+
+replace k8s.io/sample-controller => k8s.io/sample-controller v0.24.4
```

Comment:
Is there any reason why we need to add sample-cli-plugin and sample-controller in this PR?